activity/src/main/java/com/amazonaws/stepfunctions/cloudformation/activity/ResourceHandler.java

```diff
@@ -61,4 +61,13 @@
         return resultBuilder.build();
     }
 
+    protected void validateResourceArn(String resourceArn) {
```

Comment:
worth adding a doc string for this as it's inherited in a few places and it's good to be clear about the behaviour and intent of the method.

Reply:
Added